js/modules/k6/ws/ws.go

```diff
@@ -383,3 +417,21 @@ func readPump(conn *websocket.Conn, readChan chan []byte, errorChan chan error)
 		readChan <- message
 	}
 }
+
+// Wrap the raw HTTPResponse we received to a WSHTTPResponse we can pass to the user
+func wrapHTTPResponse(httpResponse *http.Response) *WSHTTPResponse {
+	wsResponse := WSHTTPResponse{
+		Status: httpResponse.StatusCode,
+	}
+
+	body, _ := ioutil.ReadAll(httpResponse.Body)
```

Comment:
Little known fact: `net/http` client functions actually return after reading the header, then it reads the body as you read it from the reader, hence why you need to fully consume the body for the connection to be reused. So, reading from the body can actually fail, and you should probably handle that case too.